dist/api_data.py

```diff
@@ -945,9 +948,9 @@ def __ge__(self, other):
         time in seconds to retain data on tiered storage on the local tier for faster read
         access''',
         min='0', max='10000'),
-    Config('object_target_size', '10M', r'''
-        the approximate size of objects before creating them on the tiered storage tier''',
-        min='100K', max='10TB'),
+    Config('object_target_size', '0', r'''
+        This option is no longer supported, retained for backward compatibility''',
+        min='0', undoc=True),
```

Comment:
I think we do want to remove this from the wiredtiger_open settings. The retained one is the one for the metadata.

Reply:
Thanks Sue! I have removed this now and will run the tests again.